glue/viewers/table/qt/data_viewer.py

```diff
@@ -260,8 +290,15 @@ def __init__(self, session, state=None, parent=None, widget=None):
         self.data = None
         self.model = None
 
-        self.ui.table.selection_changed.connect(self.selection_changed)
+        self._connection1 = connect_combo_selection(self.state, 'filter_att', self.ui.combosel_filter_att)
+        self._connection2 = connect_text(self.state, 'filter', self.ui.valuetext_filter)
+        self._connection3 = connect_checkable_button(self.state, 'regex', self.ui.bool_regex)
```

Comment:
You could perhaps use autoconnect here like we do in a lot of the other widgets?

Reply:
See https://echo.readthedocs.io/en/latest/api/echo.qt.autoconnect_callbacks_to_qt.html#echo.qt.autoconnect_callbacks_to_qt for more info